webofneeds/won-owner/src/main/java/won/owner/model/User.java

```diff
@@ -144,6 +144,8 @@ public boolean isCredentialsNonExpired() {
         return this.emailVerified || isWithinGracePeriod();
     }
 
+    public boolean isPrivateId() {return this.email.endsWith("@matchat.org");}
```

Comment:
we should change this into a won_owner-db saved attribute instead, need to update the db-schema, can be done within this PR or a sep. PR but has to be done regardless, after we added this attribute we should set the correct value of it while a signup/transfer happens, and push the attribute to the client after login/isLoggedIn as well

Reply:
I think this should be done in a separate PR, to keep this PR focused on the feature it is trying to add.